Sources/Penny/Services/AutoPingsService/DefaultPingsService.swift

```diff
@@ -10,7 +10,9 @@ import DiscordBM
 import NIOHTTP1
 
 actor DefaultPingsService: AutoPingsService {
-    
+
+    typealias Expression = S3AutoPingItems.Expression
```

Comment:
Not sure why this wasn't resulting in an error before Swift 6. `Expression` wasn't anywhere in the context but it would compile fine.